challenge_3/python/returnlove/src/majority_element.py

```diff
@@ -0,0 +1,28 @@
+"""
+To find the majority elements in a list.
+	Counts number of times each element is present in the list
+	prints all the elements which are present more than half of the list
+"""
+
+import math
+
+def find_majority_element(input_list):
+	majority_condition = math.floor(len(input_list)/2)
```

Comment:
This works but in python, you can do floor division more easily. `len(input_list)//2` has the same effect as `math.floor(len(input_list)/2)`. Whichever you prefer.